specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/examples/CosmosDBRestorableMongodbCollectionList.json

```diff
@@ -0,0 +1,31 @@
+{
+  "parameters": {
+    "api-version": "2020-06-01-preview",
+    "subscriptionId": "subid",
+    "location": "WestUS",
+    "instanceId": "98a570f2-63db-4117-91f0-366327b7b353",
+    "restorableMongodbDatabaseRid": "PD5DALigDgw="
```

Comment:
I wonder if these are actual ids of some of the Prod resources. Please use all zeros in guids and in strings you can use: "myPropertyNameObject" (Same comment everywhere else (request/response))

Reply:
spoke offline.  will keep as-is since we need to distinguish between different objects